apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V90__new_schema_rename_to_original.sql

```diff
@@ -0,0 +1,534 @@
+-- psql allows you to rename primary key
+${logic.psql-only} alter index if exists public.beneficiaries_history_new_pkey
+${logic.psql-only}    rename to beneficiaries_history_pkey;
+
+-- hsql doesn't support rename on primary key; we'll drop and re-create primary key
+${logic.hsql-only} alter table public.beneficiaries_history_new
+${logic.hsql-only}     drop constraint public.beneficiaries_history_new_pkey;
+${logic.hsql-only} alter table public.beneficiaries_history_new
+${logic.hsql-only}     add constraint beneficiaries_history_pkey primary key (bene_history_id);
+
+-- change the name of the index for bene_id
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_history_new_bene_id_idx rename to beneficiaries_history_bene_id_idx;
+  
+-- change the name of the index for bene_crnt_hic_num
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_history_new_hicn_idx rename to beneficiaries_history_hicn_idx;
+    
+-- change the name of the index for mbi_hash
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_history_new_mbi_hash_idx rename to beneficiaries_history_mbi_hash_idx;
+
+-- for hsql, we previously dropped the primary key with cascade, so we need
+-- to actually create the fk constraint
+alter table if exists public.beneficiaries_history_new
+${logic.hsql-only} add constraint beneficiaries_history_to_beneficiaries foreign key (bene_id)
+${logic.hsql-only}     references public.beneficiaries_new (bene_id);
+${logic.psql-only} rename constraint beneficiaries_history_new_to_benficiaries
+${logic.psql-only}     to beneficiaries_history_to_beneficiaries;
+
+-- ==================================
+-- medicare_beneficiaryid_history_new
+-- ==================================
+-- psql allows you to rename primary key
+${logic.psql-only} alter index if exists public.medicare_beneficiaryid_history_new_pkey
+${logic.psql-only}    rename to medicare_beneficiaryid_history_pkey;
+
+-- hsql doesn't support rename on primary key; we'll drop and re-create primary key
+${logic.hsql-only} alter table public.medicare_beneficiaryid_history_new
+${logic.hsql-only}     drop constraint public.medicare_beneficiaryid_history_new_pkey;
+${logic.hsql-only} alter table public.medicare_beneficiaryid_history_new
+${logic.hsql-only}     add constraint medicare_beneficiaryid_history_pkey primary key (bene_mbi_id);
+
+-- change the name of the index for bene_id
+alter index ${logic.psql-only} if exists 
+    public.medicare_beneficiaryid_history_new_bene_id_idx rename to medicare_beneficiaryid_history_bene_id_idx;
+
+alter table if exists public.medicare_beneficiaryid_history_new
+${logic.hsql-only} add constraint medicare_beneficiaryid_history_to_benficiaries foreign key (bene_id)
+${logic.hsql-only}     references public.beneficiaries_new (bene_id);
+${logic.psql-only} rename constraint medicare_beneficiaryid_history_new_to_benficiaries
+${logic.psql-only}     to medicare_beneficiaryid_history_to_benficiaries;
```

Comment:
typo in the new name (the old name typo seems legit but we can fix the new name here like you did for beneficiaries_history

Reply:
V91